### PR TITLE
Examples: Rename QuickHull to ConvexHull

### PR DESCRIPTION
**Pull request by Mugen87**
`ConvexHull` is a much better name than naming the class after the implemented algorithm. Also prepares `ConvexHull` and `ConvexGeometry` to become wonderful modules in the near future^^.

I'm going to note this change in the migration guide.

### CODE REVIEW

**Comment by Mugen87** [comment]
For testing:

https://raw.githack.com/Mugen87/three.js/dev33/examples/webgl_geometry_convex.html
https://raw.githack.com/Mugen87/three.js/dev33/examples/webgl_physics_convex_break.html
https://raw.githack.com/Mugen87/three.js/dev33/docs/index.html

File: docs/examples/math/convexhull/ConvexHull.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		<h1>[name]</h1>
 
 		<p class="desc">
-			General information about the Quickhull algorithm: Dirk Gregorius. March 2014, Game Developers Conference: [link:http://media.steampowered.com/apps/valve/2014/DirkGregorius_ImplementingQuickHull.pdf Implementing QuickHull].
+			Can be used to compute the convex hulls. Implementation of the Quickhull algorithm: Dirk Gregorius. March 2014, Game Developers Conference: [link:http://media.steampowered.com/apps/valve/2014/DirkGregorius_ImplementingQuickHull.pdf Implementing QuickHull].
```
Comment (WestLangley) [inline docs/examples/math/convexhull/ConvexHull.html]: Can be?

How about "A convex hull class. Implements the Quickhull algorithm by...".

Comment (Mugen87) [reply docs/examples/math/convexhull/ConvexHull.html]: Perfect!^^

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**